dom/src/main/scala/org/scalajs/dom/Element.scala

```diff
@@ -270,6 +270,22 @@ abstract class Element extends Node with NodeSelector with ParentNode with NonDo
     */
   def requestPointerLock(): Unit = js.native
 
+  /** The hasPointerCapture() method of the Element interface checks whether the element on which it is invoked has
+    * pointer capture for the pointer identified by the given pointer ID.
+    */
+  def hasPointerCapture(pointerId: Double): Boolean = js.native
+
+  /** The releasePointerCapture() method of the Element interface releases (stops) pointer capture that was previously
+    * set for a specific (PointerEvent) pointer.
+    */
+  def releasePointerCapture(pointerId: Double): Boolean = js.native
```

Comment:
```suggestion
  def releasePointerCapture(pointerId: Double): Unit = js.native
```
According to the spec `releasePointerCapture` returns undefined: https://w3c.github.io/pointerevents/#dom-element-releasepointercapture

Reply:
So it does, fixed